pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -211,6 +214,52 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 		}
 	}
 
+	scanSetting := &compliancev1alpha1.ScanSetting{}
+	if instance.SettingsRef != nil {
+		ssb := instance.DeepCopy()
+		err = r.Client.Get(context.TODO(), types.NamespacedName{Name: instance.SettingsRef.Name, Namespace: request.Namespace}, scanSetting)
+		if err != nil {
+			return reconcile.Result{}, fmt.Errorf("Failed to get ScanSetting: %s", err)
+		}
+		if scanSetting.ComplianceSuiteSettings.Suspend {
```

Comment:
would it be ok if we check `suite.spec.Suspend == true` instead of fetching the scansetting again, since we had them updated at line 211
```
err := r.applyConstraint(instance, &suite, instance.SettingsRef, log)
```